### ISSUE TITLE
mp4 [videojs is not defined]

### ISSUE THREAD

**Issue by Gigabait** [wyozi/gmod-medialib#66]
I get this error when open mp4 link.
`[HTML] Uncaught ReferenceError: videojs is not defined`

**Comment by wyozi** [comment]
Probably fixable by including the videojs library with https instead of http at https://github.com/wyozi/gmod-medialib/blob/gh-pages/mp4.html

**Comment by Gigabait** [comment]
I added https on videojs lib. But i get same error.

**Comment by wyozi** [comment]
Not sure then. Will be a while until I'll be able to look into this.

**Comment by jorjic** [comment]
Yes, switching to HTTPS fixes this for me. I can't get mp4 playing at all in the x86-64 branch. Maybe not supported.
